src/In.ProjectEKA.HipLibrary/Patient/Model/DiscoveryRequest.cs

```diff
@@ -5,11 +5,15 @@ public class DiscoveryRequest
         public PatientEnquiry Patient { get; }
 
         public string RequestId { get; }
+        public string TransactionId { get; }
+        public string Timestamp { get; }

```

Comment:
You can give some space. And use proper date type at least for the new parameters :)